arbitrum-docs/for-devs/quickstart-solidity-hardhat.md

```diff
@@ -55,13 +55,16 @@ We'll install the rest of our dependencies as we go.
    - <a data-quicklook-from='smart-contract'>Smart contracts</a> are small programs that execute transactions according to predefined rules. Ethereum's nodes host and execute smart contracts.
    - You can use smart contracts to build decentralized apps (dApps) that use Ethereum's network to process transactions and store data.
    - DApps let users carry their data and identity between applications without having to trust centralized service providers.
-   - People who run Ethereum nodes[^3] can receive rewards for processing and validating transactions on behalf of users and dApps.
+   - People who run Ethereum nodes[^3] can earn $ETH for processing and validating transactions on behalf of users and dApps.
    - These transactions can be expensive when the network is under heavy load.
  - **Arbitrum**
    - Arbitrum is a suite of L2 scaling solutions for dApp developers.
    - <a data-quicklook-from='arbitrum-one'>Arbitrum One</a> is an L2 chain that implements the <a data-quicklook-from='arbitrum-rollup'>Arbitrum Rollup</a> protocol.
    - You can use Arbitrum One to build user-friendly dApps with high throughput, low latency, and low transaction costs while inheriting Ethereum's high security standards[^4].
+  
```

Comment:
I can move these types of extracurricular edits into a separate PR if it bugs anyone :)